src/writeData/subscriptions/context/subscription.list.tsx

```diff
@@ -30,19 +37,134 @@ export const DEFAULT_CONTEXT: SubscriptionListContextType = {
   loading: RemoteDataState.NotStarted,
   change: (_id: string) => {},
   currentID: null,
+  bulletins: {},
 } as SubscriptionListContextType
 
 export const SubscriptionListContext = React.createContext<
   SubscriptionListContextType
 >(DEFAULT_CONTEXT)
 
+export interface Bulletin {
+  timestamp: number
+  message: string
+}
+
+interface BulletinsMap {
+  [key: string]: Bulletin[]
+}
+
+const sanitizeBulletin = (bulletin: string) => {
+  const patterns = [
+    'Connection to (.*) lost \\(or was never connected\\) and connection failed.',
+  ]
+  const regexObj = new RegExp(patterns.join('|'), 'i')
+  const matched = bulletin.match(regexObj)
+
+  if (!!matched?.length) {
+    bulletin = matched[0]
+  }
```

Comment:
I'm not opposed to performing specific sanitization to common bulletins, but could we also add a regex filter that can handle generic cases? Like the example [here](https://github.com/influxdata/data-acquisition/issues/329#issuecomment-1170341470) contains a large stack trace that I'd prefer to hide from the user if possible.

Reply:
For example, in the example 
```
Failed to create merged FlowFile from 1 input FlowFiles; routing originals to failure: org.influxdata.nifi.serialization.NotParsableInlineProtocolData: Not parsable data: '{ bad: data }'.\n- Caused by: java.lang.RuntimeException: unexpected date format\n- Caused by: java.text.ParseException: Unparseable date: \"}\"",
```

I think I'd like it to look like:

```
NotParsableInlineProtocolData: Not parsable data: '{ bad: data }'.\n- Caused by: java.lang.RuntimeException: unexpected date format\n- Caused by: java.text.ParseException: Unparseable date: \"}\"",
```

I'm thinking we could probably search for `org\.*.:` maybe?

Reply:
this seems to work I think: `org[^:]*(.*)`